config/tectonic-node-controller/config.go

```diff
@@ -0,0 +1,40 @@
+package tnc
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// Kind is the TypeMeta.Kind for the TNC config.
+	Kind = "TectonicNodeControllerConfig"
+	// APIVersion is the TypeMeta.APIVersion for the TNC config.
+	APIVersion = "v1"
+)
+
+// TectonicNodeControllerConfig is the config structure TNC to generate NodeConfigs
+type TectonicNodeControllerConfig struct {
+	metav1.TypeMeta       `json:",inline"`
+	HTTPProxy             string `json:"HTTPProxy"`
+	HTTPSProxy            string `json:"HTTPSProxy"`
+	NoProxy               string `json:"NoProxy"`
+	KubeletImageURL       string `json:"KubeletImageURL"`
+	KubeletImageTag       string `json:"KubeletImageTag"`
+	IscsiEnabled          string `json:"IscsiEnabled"`
+	KubeconfigFetchCmd    string `json:"KubeconfigFetchCmd"`
+	TectonicTorcxImageURL string `json:"TectonicTorcxImageURL"`
+	TectonicTorcxImageTag string `json:"TectonicTorcxImageTag"`
+	BootstrapUpgradeCl    string `json:"BootstrapUpgradeCl"`
```

Comment:
This also seems like a very out-of place option. I think this should probably be removed